From: Tetsu Yamamoto <tyamamot@redhat.com>
Date: Mon, 4 Aug 2008 15:48:46 -0400
Subject: [xen] ia64: remove regNaT fault message
Message-id: 20080804194846.8452.98141.sendpatchset@pq0-1.lab.bos.redhat.com
O-Subject: [RHEL5.3 PATCH 10/10] xen-ia64: Remove regNaT fault message
Bugzilla: 430219
RH-Acked-by: Bill Burns <bburns@redhat.com>

bz430219
# HG changeset patch
# User KUWAMURA Shin'ya <kuwa@jp.fujitsu.com>
# Date 1207901130 -32400
# Node ID af327038a43f030cdc4e111188c6585ac740d79e
# Parent  611787b6ca35fb43a811533316c799adae9cccdb
[IA64] Remove regNaT fault message

Signed-off-by: KUWAMURA Shin'ya <kuwa@jp.fujitsu.com>

diff --git a/arch/ia64/xen/faults.c b/arch/ia64/xen/faults.c
index 7083897..a127c9f 100644
--- a/arch/ia64/xen/faults.c
+++ b/arch/ia64/xen/faults.c
@@ -638,8 +638,6 @@ ia64_handle_reflection(unsigned long ifa, struct pt_regs *regs,
 		if (((isr >> 4L) & 0xfL) == 1) {
 			/* Fault is due to a register NaT consumption fault. */
 			//regs->eml_unat = 0;  FIXME: DO WE NEED THIS??
-			printk("ia64_handle_reflection: handling regNaT "
-			       "fault\n");
 			vector = IA64_NAT_CONSUMPTION_VECTOR;
 			break;
 		}
